usecases/replica/finder.go

```diff
@@ -346,7 +346,7 @@ func (f *Finder) CollectShardDifferences(ctx context.Context,
 		}, nil
 	}
 
-	replyCh, state, err := coord.Pull(ctx, One, op, "", 0)
+	replyCh, state, err := coord.Pull(ctx, One, op, "", 20*time.Second)
```

Comment:
Is `coord.Pull(ctx, One, op, "", 0)` in anything that's been released or is this just fixing a "typo" on the `merge-stable-v1.25-into-stable-v1.26` branch?

Reply:
i think we shall backport it for the past releases for 1.24 and 1.25 , so this PR actually shall be from 1.24 and then propagated to 1.26 but i just opened it like that to make sure it passes the test was introduced in 1.26

Reply:
I just checked, the `coord.Pull(ctx, One, op, "", 0)` line is only present on `merge-stable-v1.25-into-stable-v1.26`, but getting the worker context select fix backported to 1.24/1.25 sounds great